man/subscription-manager.8

```diff
@@ -388,15 +376,34 @@ command displays the current configured syspurpose
 .I preferences
 for the system.
 
+.PP
+The
+.B syspurpose
+command has subcommands for all the various syspurpose preferences and attributes:
```

Comment:
I can remember that we had to change terminology "subcommand" in the past, because this terminology is wrong. I would write something like this: "command has next level of commands for all the various system purpose preferences and attributes:"